Gemfile

```diff
@@ -73,9 +73,6 @@ gem "strong_migrations", "~> 1.2"
 # Acts as State Machine for participant and declaration states
 gem "aasm"
 
-# Pagination
-gem "pagy", "~> 5.10", ">= 5.10.1"
```

Comment:
are you sure you can remove pagy? It's used here for example https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/controllers/concerns/api_pagination.rb#L5

Reply:
Pagy is automatically pulled in by the latest version of `govuk-components`, I've switched over to using the [pagination component](https://govuk-components.netlify.app/components/pagination/).